postgresw/src/main/java/com/arcadedb/postgres/PostgresNetworkExecutor.java

```diff
@@ -347,34 +347,37 @@ private void queryCommand() {
         return;
       }
 
-      if (DEBUG)
-        LogManager.instance().log(this, Level.INFO, "PSQL: query -> %s (thread=%s)", queryText, Thread.currentThread().getId());
+//      if (DEBUG)
+//        LogManager.instance().log(this, Level.INFO, "PSQL: query -> %s (thread=%s)", queryText, Thread.currentThread().getId());
 
       final Query query = getLanguageAndQuery(queryText);
+      if (DEBUG)
+        LogManager.instance().log(this, Level.INFO, "PSQL: query -> %s ", query);
 
       final ResultSet resultSet;
-      if (queryText.startsWith("SET ")) {
-        setConfiguration(queryText);
+      if (query.query.startsWith("SET ")) {
+        setConfiguration(query.query);
         resultSet = new IteratorResultSet(createResultSet("STATUS", "Setting ignored").iterator());
-      } else if (queryText.equals("SELECT VERSION()"))
+      } else if (query.query.equals("SELECT VERSION()"))
         resultSet = new IteratorResultSet(createResultSet("VERSION", "11.0.0").iterator());
-      else if (queryText.equals("SELECT CURRENT_SCHEMA()"))
+      else if (query.query.equals("SELECT CURRENT_SCHEMA()"))
```

Comment:
The `CURRENT_SCHEMA()` function is not implemented. Should this be translated to `SELECT FROM schema:types`, or should `CURRENT_SCHEMA()` be added as a SQL function, or is this not relevant?